core/templates/dev/head/pages/topic-editor-page/editor-tab/topic-editor-tab.directive.html

```diff
@@ -11,9 +11,9 @@
       <input id="topicName" type="text" class="form-control protractor-test-topic-name-field"
              ng-disabled="!topicRights.canEditTopic()"
              ng-model="editableName" ng-blur="updateTopicName(editableName)"
-             placeholder="Enter a name for the topic." maxlength="20" ng-trim="false">
-      <span class="form-text" style="font-size: smaller; color: red;" ng-if="editableName.length === 20">
-        <em>Please use at most 20 characters for the topic name.</em>
+             placeholder="Enter a name for the topic." maxlength="35" ng-trim="false">
```

Comment:
Length limits should also be verified in the backend; do we do so?

(In general, all validation checks should be applied in both the backend and frontend.)

Reply:
Done for topic name and description. Check for abbreviated name was already done.